packages/jest-jasmine2/src/index.js

```diff
@@ -100,6 +101,10 @@ async function jasmine2(
       expand: globalConfig.expand,
     });
 
+  if (globalConfig.errorOnDeprecated) {
+    installErrorOnPrivate(environment.global);
+  }
```

Comment:
I wonder if we should move this outside of jest jasmine. If user changes runner to circus, then depreciation errors will be gone, and only cryptic errors like "cannot call function on undefined" will surface

Reply:
Any idea where this could live that it would be shared by both?